linker/linker_test.go

```diff
@@ -1013,7 +1032,7 @@ func TestLinkerValidation(t *testing.T) {
 					  extensions 1 to 100;
 					  message b {
 					    message c {
-					      extend a { repeated int32 i = 1; repeated float f = 2; }
```

Comment:
This addresses a comment in previous PR: https://github.com/bufbuild/protocompile/pull/87#discussion_r1082722319